{{cookiecutter.plugin_name}}/src/{{cookiecutter.module_name}}/_writer.py

```diff
@@ -25,7 +40,23 @@ def write_single_image(path: str, data: Any, meta: dict) -> List[str]:
 
 
 def write_multiple(path: str, data: List[FullLayerData]) -> List[str]:
-    """Writes multiple layers of different types."""
+    """Writes multiple layers of different types.
+    
+    Parameters
+    ----------
+    path : str
+           A string path indicating where to save the image file.
+    data : A layer tuple.
```

Comment:
list of!

Reply:
Aaargh
Thanks! This is why we need good doctrings 😆